website/index.html

```
@@ -0,0 +1,577 @@
+        btn.type = 'button'
+        btn.textContent = s.label
+        btn.addEventListener('click', () => renderSnippet(i))
+        tabs.appendChild(btn)
+      })
+      renderSnippet(0)
+
+      // Nav scroll style
+      const nav = document.getElementById('nav')
+      window.addEventListener(
+        'scroll',
+        () => {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)